auxprogs/compileSpliceCands/Makefile

```
@@ -1,8 +1,9 @@
 compileSpliceCands : compileSpliceCands.o list.h list.o
-	gcc $(LDFLAGS) -o compileSpliceCands compileSpliceCands.o list.o;
+	$(CC) -o compileSpliceCands compileSpliceCands.o list.o $(CFLAGS) $(LDFLAGS);
```

```suggestion
	$(CC) $(CFLAGS)	$(CPPFLAGS) $(LDFLAGS) -o compileSpliceCands compileSpliceCands.o list.o
```
-  $(CPPFLAGS) missing?
-  why $(CFLAGS) and $(LDFLAGS) at the line end and not direct after $(CC) like in the other Makefiles
- remove semicolon from end?